payments-ui-core/src/main/resources/lpms.json

```diff
@@ -1057,5 +1057,32 @@
         }
       }
     }
+  },
+  {
+    "type": "multibanco",
+    "async": true,
+    "fields": [
+      {
+        "type": "email",
+        "api_path": {
+          "v1": "billing_details[email]"
+        }
+      }
+    ],
+    "next_action_spec": {
+      "confirm_response_status_specs": {
+        "requires_action": {
+          "type": "multibanco_display_details"
+        }
+      },
+      "post_confirm_handling_pi_status_specs": {
+        "succeeded": {
+          "type": "finished"
+        },
+        "requires_action": {
+          "type": "canceled"
+        }
+      }
+    }
```

Comment:
Wasn't clear on whether we actually need this given #8053

Reply:
I've been leaving the JSON exactly as it is on the server to try to make things consistent and not confusing.

Reply:
On iOS leaving this next_action_spec would probably make handling the next action fail FWIW.

Reply:
Removed this after Jay and I chatted. It's not in the server spec anyway